versioned_docs/version-v0.15/UserManuals/DORA.md

```diff
@@ -64,113 +66,123 @@ Calculating DORA metrics requires three key entities: **changes**, **deployments
+   ![project1](/img/ConfigUI/project1.png)
+   ![](https://i.imgur.com/lpPRZ6v.png)
 
 3. Select Jira boards and GitHub repos to collect, click `Next Step`
-![](https://i.imgur.com/Ko38n6J.png)
+   ![](https://i.imgur.com/Ko38n6J.png)
 
 4. Click `Add Transformation` to configure for DORA metrics
-![](https://i.imgur.com/Lhcu2DE.png)
+   ![](https://i.imgur.com/Lhcu2DE.png)
 
 5. To make it simple, fields with a ![](https://i.imgur.com/rrLopFx.png) label are DORA-related configurations for every data source. Via these fields, you can define what are "incidents" and "deployments" for each data source. After all data connections have been configured, click `Next Step`
+
    - This team uses Jira issue types `Crash` and `Incident` as "incident", so choose the two types in field "incident". Jira issues in these two types will be transformed to "incidents" in DevLake.
    - This team uses the GitHub action jobs named `deploy` and `build-and-deploy` to deploy, so type in `(?i)deploy` to match these jobs. These jobs will be transformed to "deployments" in DevLake.
-   ![](https://i.imgur.com/1JZA2xn.png)
-   
+     ![](https://i.imgur.com/1JZA2xn.png)
+
    Note: The following example shows where to find GitHub action jobs. It's easy to mix them up with GitHub workflows.
    ![](https://i.imgur.com/Y2hchEh.png)
-   
 
 6. Choose sync frequency, click 'Save and Run Now' to start data collection. The time to completion varies by data source and depends on the volume of data.
-![](https://i.imgur.com/zPkfzGr.png)
+   ![](https://i.imgur.com/zPkfzGr.png)
 
 For more details, please refer to our [blueprint manuals](https://devlake.apache.org/docs/UserManuals/ConfigUI/Tutorial).
 
 ### Collect CircleCI data via `webhook`
 
 Using CircleCI as an example, we demonstrate how to actively push data to DevLake using the Webhook approach, in cases where DevLake doesn't have a plugin specific to that tool to pull data from your data source.
 
-7. Visit "Data Connections" page in config-ui and select "Issue/Deployment Incoming Webhook".
+7. Go to the 'Data Connections' page. Create a webhook.
+
+![webhook-add-data-connections](/img/ConfigUI/webhook-add-data-connections.png)
 
-8. Click "Add Incoming Webhook", give it a name, and click "Generate POST URL". DevLake will generate URLs that you can send JSON payloads to push `deployments` and `incidents` to Devlake. Copy the `Deployment` curl command.
-![](https://i.imgur.com/jq6lzg1.png)
-![](https://i.imgur.com/jBMQnjt.png)
+We recommend that you give your webhook connection a unique name so that you can identify and manage where you have used it later.
+
+8. Create a Project first, choose Incoming Webhooks, then you can `Add a Webhook` or `Select Existing Webhooks`. And click "Generate POST URL". DevLake will generate URLs that you can send JSON payloads to push `deployments` and `incidents` to Devlake. Copy the `Deployment` curl command.
```

Comment:
`Add a Webhook` here so that step 7 is not required?

Reply:
yes, I think the user can add a webhook in project directly